scripts/lib/install

```diff
@@ -162,6 +175,14 @@ if [ "$EXTERNAL_HOST" = zulip.example.com ] \
     exit 1
 fi
 
+if [ -z "$POSTGRESQL_DATABASE_NAME" ]; then
+    POSTGRESQL_DATABASE_NAME="zulip"
+fi
+
+if [ -z "$POSTGRESQL_DATABASE_USER" ]; then
+    POSTGRESQL_DATABASE_USER="zulip"
+fi
+
```

Comment:
I'm a little on the fence about always writing out the default values of `zulip` into the config.
1. It means we have to carefully also write them out during upgrades (which is missing here)
2. It means we can put the default values once here, and not _everywhere_ that accesses them
3. It clutters the configuration file with values that are not interesting to the sysadmin; we try to keep the configuration file short and poignant, where every value you see matters

I think, all in all, that I'd rather drop this hunk, so we don't write out values of `zulip` explicitly, and change the places that access it to include an explicit default of `zulip`.  We don't expect to change that default ant any time, so the utility of point (2) is not as much, so points (1) and (3) win out, I think.

Reply:
@alexmv whilst I certainly agree with idea here, as far as I am aware `crudini` doesn't allow us to set an explicit default. So not sure there is another way round it.